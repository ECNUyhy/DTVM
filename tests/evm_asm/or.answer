000000000000000000000000000000000000000000000000000000000000003F