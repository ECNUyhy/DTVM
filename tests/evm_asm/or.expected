status: success
error_code: 0
stack: []
memory: 000000000000000000000000000000000000000000000000000000000000003F
storage: {}
transient_storage: {}
return: 000000000000000000000000000000000000000000000000000000000000003F
events: []
